src/ipc/client.rs

```diff
@@ -511,6 +512,25 @@ fn print_output(output: Output) -> anyhow::Result<()> {
     let serial = serial.as_deref().unwrap_or("Unknown");
     println!(r#"Output "{make} {model} {serial}" ({name})"#);
 
+    let print_qualifier = |is_preferred: bool, is_current: bool, is_custom_mode: bool| {
+        let mut qualifier = Vec::new();
+        if is_preferred {
+            qualifier.push("preferred");
+        };
```

Comment:
This changes behavior. It prints preferred before current, whereas the original code originally printed current first as it's more important.